yang/frr-gmp.yang

```diff
@@ -55,31 +55,48 @@ module frr-igmp {
      (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
      OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.";
 
-  revision 2019-11-06 {
+  revision 2021-11-22 {
     description
       "Initial revision.";
     reference
       "RFC 2236: IGMP v2.
-       RFC 3376: IGMP v3.";
+       RFC 3376: IGMP v3.
+       RFC 2710: MLD.";
```

Comment:
for consistency, please add 3810 for MLDv2 (users could be confused otherwise about what we support)

Reply:
Done